public/javascripts/commonFunctions.js

```diff
@@ -1,4 +1,9 @@
 $(document).ready(function(){
 	$(".navbar .nav-item.active").removeClass('active');
 	$('.navbar .nav-item a[href="' + location.pathname + '"]').closest('li').addClass('active');
-});
\ No newline at end of file
+
+	if (!document.cookie.split('; ').find(row => row.startsWith("closedSupportToast"))) {
+    $(".toast").toast("show");
+  }
+	$("button#closeToast").on("click", () => { document.cookie = "closedSupportToast=true; expires=Fri, 31 Dec 2020 23:59:59 GMT"; })
```

Comment:
Hard-coded date will stop working after some time. Make it dynamic (a month/3 or 6 months)